bpf/bpf_lxc.c

```diff
@@ -685,9 +687,11 @@ static __always_inline int handle_ipv4_from_lxc(struct __ctx_buff *ctx,
 		else
 			return ret;
 	}
-#else
-	goto pass_to_stack;
 #endif
+	if (is_defined(ENABLE_REDIRECT_NEIGH))
+		return redirect_neigh(ct_state.ifindex ? :
```

Comment:
Just to double check, but does the IPv4 TTL get decremented after calling `redirect_neigh()`?